scripts/experiments_tabular/mogaal_run.sh

```diff
@@ -6,12 +6,12 @@
 MAX_SEED=$1
 DATASET=$2
 
-module load Julia/1.4.1-linux-x86_64
+module load Julia/1.5.1-linux-x86_64
 module load Python/3.8.2-GCCcore-9.3.0
 
 # load virtualenv
-source ${HOME}/sklearn-env/bin/activate
-export PYTHON="${HOME}/sklearn-env/bin/python"
+source ${HOME}/ad/bin/activate
```

Comment:
This is potentially dangerous if you run two scripts that are using different python environments - PyCall is gonna be compiled over and over again and things might crash.